app/src/main/java/grinnell/appdev/edu/lyles/MenuItem.java

```diff
@@ -0,0 +1,68 @@
+package grinnell.appdev.edu.lyles;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Shelby on 2/3/2017.
+ */
+
+public class MenuItem {
+    private String title;
+    private double price;
+    private String imageUrl;
+    private String details;
+
+    public MenuItem() {
+        title = null;
+        price = 0;
+        imageUrl = null;
+        details = null;
+    }
+
+    public MenuItem(JSONObject jsonObject) {
+        try {
+            this.title = jsonObject.getString("title");
```

Comment:
I advise that you create a class with constants and put all of these keys in there.